### PR TITLE
Adds calculation of fee

### PR DESCRIPTION
**Pull request by skopczynska**

### CODE REVIEW

File: 01-CodeReviewEx/Shop-Ex/Account.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shop_Ex
+{
+    public class Account
+    {
+        public double principal, rate; int daysActive, accountType;
```
Comment (ajjsha) [inline 01-CodeReviewEx/Shop-Ex/Account.cs:11]: move variables of new type to next line for better visibility

File: 01-CodeReviewEx/Shop-Ex/Program.cs [hunk 2]
```diff
@@ -10,11 +10,30 @@ internal class Program
     {
         static void Main(string[] args)
         {
+            Account[] myAccounts = new Account[10];
+            Account a = new Account();
+            a.principal = 1.0;
+            a.rate = 0.5;
+            myAccounts[0] = a;
+            a = new Account() { principal = 1.5, rate = 0.75 };
```
Comment (ajjsha) [inline 01-CodeReviewEx/Shop-Ex/Program.cs:18]: lack of consistency in creating new variables; also object with the same variable name created twice

File: 01-CodeReviewEx/Shop-Ex/Program.cs [hunk 3]
```diff
@@ -10,11 +10,30 @@ internal class Program
     {
         static void Main(string[] args)
         {
+            Account[] myAccounts = new Account[10];
+            Account a = new Account();
```
Comment (wronekkrzysztof) [inline 01-CodeReviewEx/Shop-Ex/Program.cs:14]: standarize of constraction use

File: 01-CodeReviewEx/Shop-Ex/Program.cs [hunk 4]
```diff
@@ -10,11 +10,30 @@ internal class Program
...
 
-        
+        public static double calculateFee(Account[] accounts)
+        {
+            double totalFee = 0.0;
+            Account account;
+            for (int i = 0; i < accounts.Length; i++)
+            {
+                account = accounts[i];
+                if (account.accountType == Account.PREMIUM || account.accountType == Account.PREMIUM_PLUS) totalFee += .0125 * ( // 1.25% broker's fee
```
Comment (wronekkrzysztof) [inline 01-CodeReviewEx/Shop-Ex/Program.cs:32]: more readable math expresions

**Review by wronekkrzysztof — COMMENTED** [review]
Account

File: 01-CodeReviewEx/Shop-Ex/Program.cs [hunk 5]
```diff
@@ -10,11 +10,30 @@ internal class Program
...
         {
+            Account[] myAccounts = new Account[10];
+            Account a = new Account();
+            a.principal = 1.0;
+            a.rate = 0.5;
+            myAccounts[0] = a;
+            a = new Account() { principal = 1.5, rate = 0.75 };
             
+            double fee = calculateFee(myAccounts);
+            Console.WriteLine(fee);
```
Comment (ajjsha) [inline 01-CodeReviewEx/Shop-Ex/Program.cs:22]: empty line not needed

**Review by ajjsha — CHANGES_REQUESTED** [review]

**Pull request closed without merging**